src/ScottBrady.IdentityModel/Tokens/EdDsa.cs

```diff
@@ -9,7 +10,7 @@
 
 namespace ScottBrady.IdentityModel.Tokens;
 
-public class EdDsa
+public class EdDsa: AsymmetricAlgorithm
```

Comment:
Looks like this one might have some knock on effects that I'd like to test before I release. I'll merge this in though and run with it.